README.md

```diff
@@ -675,26 +662,25 @@ $itemCallable = function (CliMenu $menu) {
 $menu = (new CliMenuBuilder)
     ->setWidth(150)
     ->addStaticItem('Below is a SplitItem')
-    ->addSplitItem()
-        ->setGutter(5)
-        ->addSubMenu('Sub Menu on a split item')
-            ->setTitle('Behold the awesomeness')
-            ->addItem('This is awesome', function() { print 'Yes!'; })
-            ->end()
-        ->addItem('Item 2', $itemCallable)
-        ->addStaticItem('Item 3 - Static')
-        ->end()
+    ->addSplitItem(function (SplitItemBuilder $b) use ($itemCallable) {
+        $b->setGutter(5)
+            ->addSubMenu('Sub Menu on a split item', function (CliMenuBuilder $b) {
+                $b->setTitle('Behold the awesomeness')
+                    ->addItem('This is awesome', function() { print 'Yes!'; });
+            })
+            ->addItem('Item 2', $itemCallable)
+            ->addStaticItem('Item 3 - Static');
+    })
     ->build();
 
 $menu->open();
 ```
 
 There are a few things to note about the syntax and builder process here:
 
-1. There is no argument to `addSplitItem`.
-2. `addSplitItem` returns an instance of `SplitItemBuilder`, the main menu will take care of building all split items (and any submenu they might contain).
-3. You can call `addItem`, `addSubMenu` and `addStaticItem` on the `SplitItemBuilder`.
-4. Like when building submenus, you can call `end()` on the Split Item `SplitItemBuilder` instance to get the parent `CliMenuBuilder` back again. This is useful for chaining.
+1. The first parameter to `addSubMenu` is a closure, which will be invoked with a new instance of `SplitItemBuilder` which you can use to add items to the split item.
+2. You can call `addItem`, `addSubMenu` and `addStaticItem` on the `SplitItemBuilder`. 
+3. `SplitItemBuilder` has a fluent interface so you can chain method calls.
```

Comment:
Add a note that like the addSubMenu, the closure used to build the splititem is also binded with the `SplitItemBuilder` blablabla